Add a Python3 version

markmmm opened the pull request:
I changed variable names to lowercase - but I guess could have left them too. Tests pass!

Additionally I believ I have kept the code compatible with lower versions of Python 3 (by avoiding using dataclasses)

mmm-8192 committed:
Add a Python3 version

I changed variable names to lowercase - but I guess could have left them too.
Tests pass!

Additionally I believ I have kept the code compatible with lower versions of
Python 3 (by avoiding using dataclasses)

codecop commented:
Thank you. Keeping it open for older versions of Python is great. This is legacy code after all ;-)
I propose some minor modifications 

- There is no need for the license file, if you keep it you can add your own name. 
- There are references to PHP in the readme, please update readme to reflect Python and how to run tests.
- Optional but great if you can add a github workflow (.github folder) as for the other languages. Similar to [this one](https://github.com/codecop/Parrot-Refactoring-Kata/blob/github-build/.github/workflows/Python.yml) but no pip and requirements etc.

mmm-8192 committed:
Update Readme and remove LICENSE

markmmm committed:
Add github action for Python kata

markmmm committed:
Remove redundant directory

We are already in `Python` directory - no need to have `Python` in the path again

markmmm commented:
>     * Optional but great if you can add a github workflow (.github folder) as for the other languages. Similar to [this one](https://github.com/codecop/Parrot-Refactoring-Kata/blob/github-build/.github/workflows/Python.yml) but no pip and requirements etc.

Thanks for that push (and for the very useful reference) - I now have created my first github action! :)

codecop commented:
Thank you.
